src/main/java/Main.java

```diff
@@ -33,7 +33,7 @@ private Pane loadMainPane() throws IOException {
         Pane mainPane = (Pane) loader.load(getClass().getResourceAsStream(VistaNavigator.MAIN));
         MainController mainController = loader.getController();
         VistaNavigator.setMainController(mainController);
-        VistaNavigator.loadVista(VistaNavigator.VISTA_1);
+        //VistaNavigator.loadVista(VistaNavigator.VISTA_1);
         return mainPane;
     }
 
```

Comment:
shouldn't comment it out, this is the page that will show to user (contain login info etc...) need to see that it work with you changes.

Reply:
If u could explain what is the difference between line 33 and 36.
In the original example, it's loaded a page combined with Header from MAIN Body and VISTA1

Reply:
yes, and this is the way to do it... you only change "part" of tha page.